[2025-01-26T21:30:04.777+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T21:30:04.826+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_processing.schema_create scheduled__2025-01-26T21:28:00+00:00 [queued]>
[2025-01-26T21:30:04.859+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_processing.schema_create scheduled__2025-01-26T21:28:00+00:00 [queued]>
[2025-01-26T21:30:04.860+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-26T21:30:04.894+0000] {taskinstance.py:2889} INFO - Executing <Task(SQLExecuteQueryOperator): schema_create> on 2025-01-26 21:28:00+00:00
[2025-01-26T21:30:04.914+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1115) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-26T21:30:04.917+0000] {standard_task_runner.py:72} INFO - Started process 1125 to run task
[2025-01-26T21:30:04.920+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_processing', 'schema_create', 'scheduled__2025-01-26T21:28:00+00:00', '--job-id', '377', '--raw', '--subdir', 'DAGS_FOLDER/DAG_etl.py', '--cfg-path', '/tmp/tmp0ftbc794']
[2025-01-26T21:30:04.926+0000] {standard_task_runner.py:105} INFO - Job 377: Subtask schema_create
[2025-01-26T21:30:05.090+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_processing.schema_create scheduled__2025-01-26T21:28:00+00:00 [running]> on host a664bcf281bf
[2025-01-26T21:30:05.838+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_processing' AIRFLOW_CTX_TASK_ID='schema_create' AIRFLOW_CTX_EXECUTION_DATE='2025-01-26T21:28:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-26T21:28:00+00:00'
[2025-01-26T21:30:05.843+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T21:30:05.891+0000] {sql.py:278} INFO - Executing: 
    create schema if not exists nds;
    create schema if not exists dds;
    create schema if not exists stg;
[2025-01-26T21:30:05.914+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T21:30:05.933+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T21:30:05.947+0000] {sql.py:544} INFO - Running statement: 
    create schema if not exists nds;
    create schema if not exists dds;
    create schema if not exists stg;, parameters: None
[2025-01-26T21:30:05.985+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T21:30:05.986+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_processing, task_id=schema_create, run_id=scheduled__2025-01-26T21:28:00+00:00, execution_date=20250126T212800, start_date=20250126T213004, end_date=20250126T213005
[2025-01-26T21:30:06.071+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-26T21:30:06.099+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
